jax_distribution/wrappers/jumanji.py

```diff
@@ -24,6 +24,24 @@
 from jumanji.wrappers import Wrapper
 
 
+class ObservationGlobalState(NamedTuple):
+    """The observation that the agent sees.
+    agents_view: the agents' view of other agents and shelves within their
+        sensor range. The number of features in the observation array
+        depends on the sensor range of the agent.
+    action_mask: boolean array specifying, for each agent, which action
+        (up, right, down, left) is legal.
+    global_state: the global state of the environment, which is the
+        concatenation of the agents' views.
+    step_count: the number of steps elapsed since the beginning of the episode.
+    """
+
+    agents_view: chex.Array  # (num_agents, num_obs_features)
+    action_mask: chex.Array  # (num_agents, 5)
+    global_state: chex.Array  # (num_agents * num_obs_features,)
+    step_count: chex.Array  # ()
```

Comment:
Small fix here, the shape should be `# (num_agents, )`

Reply:
yes and also I need to fix global_state comment `(num_agents,num_agents* num_obs_features)`

Reply:
Shouldn't global state only be a single value?

Reply:
Ah, I had a quick look. We duplicate the global state per agent. I think this should be fine, we can keep it as is :)

Reply:
so this will help in getting things standardized however if you think that this will cause any issues if we extend the number of agents or/and envs I can make some changes to it